src/components/HomePage/index.js

```diff
@@ -86,29 +95,34 @@ function HomePage({ background = "white", textColor = "black" }) {
               <ListItem button>
                 <ListItemText primary="#css" />
               </ListItem>
-                        </List>
-                   
-                  </Grid>
-                </Grid>
-
-              {window.innerWidth > 750 && <Grid
-                  container
-                  className={classes.leftSideCard}
-                  direction="column"
-                  style={{
-                      width: "100%",
-                      overflow: "auto",
-                      maxHeight: "25rem",
-                      backgroundColor: "transparent",
-                      border: "none",
-                      boxShadow: "none"
-                  }}
-              >
+            </List>
+          </Grid>
+        </Grid>
 
-                  <Grid item className={classes.outerSideBar}>
-                      <SideBar open={openMenu} toggleSlider={toggleSlider} />
-                  </Grid>
-              </Grid>}
+        {window.innerWidth > 750 && (
```

Comment:
can you explain what is the purpose of using `window.innerWidth` instead using [material UI breakpoints](https://mui.com/material-ui/customization/breakpoints/)?

Reply:
for conditional rendering using a mui breakpoint of 750px, I had to declare it in the CSS style of the class as there is no breakpoint provided for 750px by mui. On the other hand, the window.innerWidth was allowing me to give conditions directly. There is no other purpose behind doing it. If window.innerWidth does not seem feasible to you please let me know. I would change according to your suggestion.

Reply:
@Niweera Thank you, I will do!

Reply:
cool